requirements.txt

```diff
@@ -1,17 +1,18 @@
-argschema<2.0.0
 allensdk
+argschema
 dictdiffer
-h5py==2.10.0
-marshmallow==3.0.0rc6
-matplotlib>=1.4.3
+h5py
+marshmallow
+matplotlib
 methodtools
-numpy>=1.15.4,<1.19.0
-pandas>=0.25.1,<=0.25.3
+numpy
+pandas
 pg8000
 pillow
-pyabf<2.3.0
-pynwb>=1.3.2,<2.0.0
-pyYAML<6.0.0
-scipy>=0.15.1
-simplejson>=3.10.0
+pyabf
+pynwb==2.2.0
```

Comment:
Have you tried with `pynwb~=2.2`? For external users having this pinned too strictly may cause pain points?

Reply:
Yes, later versions (2.3.0+) start to fail a couple of tests, so 2.2.0 is the maximum for now without further changes.